app/views/members/dues/_form.html.haml

```diff
@@ -8,7 +8,7 @@
     "$25.00 USD Monthly" => "medium_monthly",
     "$20.00 USD Monthly" => "20_monthly",
     "$15.00 USD Monthly" => "15_monthly",
-    "$10.00 USD Monthly" => "small_monthly" }, "30_monthly")
+    "$10.00 USD Monthly" => "small_monthly" }, @current_plan || "30_monthly")
```

Comment:
Very optional: It would be nice to have only one copy of this hash list of plan names, and reuse it both in `dues_controller.rb` and here. Maybe `dues_controller.rb` could use the hash that's being returned by `def plans` to generate a hash that then gets used here in the form.